scripts/install-service.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -e 
+
+# Define data directory to be used for application config file.
+DATA_DIR=${DATA_DIR:-${HOME}/.poglink}
+
+# Change to script directory for predictable behaviour
+cd "$(dirname "$0")"
+
+# Check whether virtual environment is active
+if [ -z "${VIRTUAL_ENV}" ] && [ "${FORCE}" != "true"  ]; then
+    echo "Warning: Virtual environment not detected! It is highly recommended to install this program to run under a virtual environment. Please activate your virtual environment and run this script again."
+    echo "If you're certain you'd like to continue without a virtual environment, you can run again with 'FORCE=true'." 
+    exit 1
+fi
+
+# Install poglink python package
+pip install poglink
```

Comment:
Didn't want to pin a specific version or force an update; I feel like that should be up to the user to decide if they want a specific version or want to update.